pkg/looker/resource_project.go

```diff
@@ -0,0 +1,82 @@
+package looker
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	apiclient "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
+)
+
+func resourceProject() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceProjectCreate,
+		ReadContext:   resourceProjectRead,
+		DeleteContext: resourceProjectDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringDoesNotContainAny(" "),
+			},
+		},
+	}
+}
+
+func resourceProjectCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	client := m.(*apiclient.LookerSDK)
+	// Set client to dev workspace
+	if err := updateApiSessionWorkspaceId(client, "dev"); err != nil {
+		return diag.FromErr(err)
+	}
+
+	// Create the project
+	projectName := d.Get("name").(string)
+	writeProject := apiclient.WriteProject{
+		Name:            &projectName,
+	}
+	project, err := client.CreateProject(writeProject, nil)
```

Comment:
Cloud you add `[DEBUG]` log before api call in Create, Read, Update, and Delete?

For example,
https://github.com/hirosassa/terraform-provider-looker/blob/master/pkg/looker/resource_user_attribute.go#L75